### PR TITLE
NGLS-3202 Superset: Add indicator that download is in progress for Dashboards

### PR DESCRIPTION
**Pull request by EduardoSanday**
After the user clicks on options to download the dashboard, a generic message will be pop up with the following text:
"Preparing file for download, please wait."

### CODE REVIEW

File: superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx [hunk 1]
```diff
@@ -197,6 +198,9 @@ class HeaderActionsDropdown extends React.PureComponent {
           menu.style.visibility = 'visible';
         });
         this.props.logEvent?.(LOG_ACTIONS_DASHBOARD_DOWNLOAD_AS_IMAGE);
+        this.props.addInfoToast(
+          t('Image download takes a long time, please wait.'),
```
Comment (gionatta) [inline superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx]: I think we can use a generic message to indicate a file is being prepared for download. The message would be the same for image and pdf.
Eg: "Preparing file for download, please wait."

File: superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx [hunk 2]
```diff
@@ -215,6 +219,9 @@ class HeaderActionsDropdown extends React.PureComponent {
           menu.style.visibility = 'visible';
         });
         this.props.logEvent?.(LOG_ACTIONS_DASHBOARD_DOWNLOAD_AS_PDF);
+        this.props.addInfoToast(
+          t('PDF download takes a long time, please wait.'),
```
Comment (gionatta) [inline superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx]: See above comment.

**Review by gionatta — CHANGES_REQUESTED** [review]

**Review by gionatta — APPROVED** [review]

**Review by gfieremans — COMMENTED** [review]
Shouldn't this be for charts as well?

**Review by gfieremans — APPROVED** [review]

**Comment by EduardoSanday** [comment]
> Shouldn't this be for charts as well?

Yes, I opened a separate story to do this implementation.
[NGLS-3279](https://jira.dev.global-intra.net:8443/browse/NGLS-3279)

**Pull request merged**